src/resources/views/corporation/ledger/reprocessingfeebymonth.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('web::corporation.ledger.layouts.view', ['sub_viewname' => 'bountyprizesbymonth'])
+
+@section('title', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+@section('page_header', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+
+@section('ledger_content')
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Available Ledgers</h3>
+    </div>
+    <div class="panel-body">
+
+      @foreach ($bountyprizes->chunk(3) as $chunk)
```

Comment:
`bountyprizes` will not exist here according to your controller. I suppose it should be `reprocessingfeetotals`.